question-servers/freeform.js

```diff
@@ -199,7 +217,7 @@ module.exports = {
             const courseErr = new Error('Invalid state before ' + phase + ': ' + checkErr);
             courseErr.fatal = true;
             courseErrs.push(courseErr);
-            return callback(null, courseErrs, data, '', fileData);
+            return callback(null, courseErrs, isGetDependencies ? elementDependencies : data, '', fileData);
```

Comment:
Yeah, I was doing this just to avoid an ever-growing list of arguments as we add phases and functionality. I'd err on the side of doing that, but I'd be happy to change it to another argument if you prefer that.